backend/src/main/java/de/bund/digitalservice/ris/search/service/helper/PaginationParamsConverter.java

```diff
@@ -63,4 +65,12 @@ private static boolean isSupportedField(
       case CASE_LAW -> SortingUtils.caseLawSortFields.contains(sort);
     };
   }
+
+  private static Sort getDefaultSort(MappingDefinitions.ResolutionMode resolutionMode) {
+    return switch (resolutionMode) {
+      case ALL -> Sort.by(Sort.Direction.DESC, "DATUM");
+      case NORMS -> Sort.by(Sort.Direction.DESC, Norm.Fields.NORMS_DATE);
+      case CASE_LAW -> Sort.by(Sort.Direction.DESC, CaseLawDocumentationUnit.Fields.DECISION_DATE);
+    };
+  }
```

Comment:
Do we have more code that distinguishes between document kinds? If yes then we should maybe consider to have a special configuration classes for this?

Reply:
Good call, I've moved the code to a renamed class `SortingDefinitions`, placed next to `MappingDefinitions`